docs/source/configuration/tracing.mdx

```diff
@@ -0,0 +1,189 @@
+---
+title: Tracing in the Apollo Router
+description: Collect tracing information
+---
+
+import { Link } from "gatsby";
+
+The Apollo Router supports [OpenTelemetry](https://opentelemetry.io/), with exporters for:
+
+* [Jaeger](https://www.jaegertracing.io/)
+* [Zipkin](https://zipkin.io/)
+* [Datadog](https://www.datadoghq.com/)
+* [OpenTelemetry Protocol (OTLP)](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md) over HTTP or gRPC.
+
+The Apollo Router generates spans that include the various phases of serving a request and associated dependencies. This is useful for showing how response time is affected by:
+
+* Sub-request response times
+* Query shape (sub-request dependencies)
+* Apollo Router post-processing
+
+Span data is sent to a collector such as [Jaeger](https://www.jaegertracing.io/), which can assemble spans into a gantt chart for analysis.
+
+> To get the most out of distributed tracing, _all_ components in your system should be instrumented.
+
+## Common configuration
+
+### Trace config
+The `trace_config` section contains common configuration that will be used in the for all exporters. It is optional and will fall back on env variables as specified by the [OpenTelemetry spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md) if `service_name` is not set.
+
+```yaml title="router.yaml"
+telemetry:
+  tracing:
+    trace_config:
+      service_name: "${MY_ENV_VARIABLE:router}"
+      service_namespace: "apollo"
+      # Optional. Either a float between 0 and 1 or 'AlwaysOn' or 'AlwaysOff'
+      sampler: AlwaysOn
```

Comment:
I wonder if we should stay consistent and have snake_case everywhere ?

Reply:
Yeah, I missed this first time round. But this is now fixed.